CONTRIBUTING_JS.md

```diff
@@ -132,9 +132,14 @@ Always run tests before pushing and PR'ing your code.
 
 #### Supported Versions
 
-IPFS projects support both the Current and Active LTS versions of Node.js.  Please see [nodejs.org](https://nodejs.org/) for what these currently are.
+The IPFS JavaScript projects follow the LTS of Node.js and respective npm version that gets installed with Node.js. Please consult [nodejs.org](https://nodejs.org/) for LTS timeline. Today these are:
 
-Browser code may be transpiled using Babel to take advantage of the latest JavaScript features.
+```
+» node -v
```

Comment:
Perhpas doing a `date -u +"%Y-%m-%dT%H:%M:%SZ"` first, so that it is clear what "today" means?

Reply:
done